packages/bindgen/src/templates/typescript.ts

```diff
@@ -143,18 +143,19 @@ export function generateTypeScript({ spec: rawSpec, file }: TemplateContext): vo
     }
   `);
 
-  const js = file("native.js", "eslint");
+  const js = file("native.mjs", "eslint");
   js("// This file is generated: Update the spec instead of editing this file directly");
   js(`
-    import bindings from 'bindings';
-    const nativeModule = bindings("realm.node")
+    import { createRequire } from 'node:module';
+    const require = createRequire(import.meta.url);
+    const nativeModule = require("./realm.node");
```

Comment:
I couldn't tell if it would be better to hide the fact that this is a `require()` function from the bundlers or not. Either way, easy enough to change.